### DISCUSSION TITLE
GitHub is looking for feedback on GitHub Projects Insights!

### DISCUSSION THREAD

**Discussion by chiedo** [community/community#41867] [Projects and Issues] [answered]
👋
The Planning and Tracking team at GitHub is eager to build features in GitHub Projects that help you visualize your productivity so you can make data-driven decisions!

We’re looking to enhance the charting and insights feature-set and would love to hear from you if you have feedback to share about your experience and your wishes.

Please share with us - [in this quick survey](https://forms.gle/7RH6eYVNBZ3zBZ1r9). 

Thank you!

🚀 ✨

**Comment by dirtybike** [comment]
Can we have the ability to compare one project turndown to another? That function and the function to duplicate another project would be sooo helpful!! I don't want to beg for the latter but pleasssssse make it possible to duplicate projects because it will decidedly heal a big pain: Today, custom columns added in a previous project need to be manually added for new projects and any tickets that were not completed in the previous project (or sprint) need to be manually moved over to the new project.

**Comment by natalierjackson** [comment, marked as answer]
Thank you for all your valuable feedback. This survey is now closed.

We take great pride in our work, and our sincerest hope is that it helps each of our customers accomplish their goals. Your feedback has demonstrated that it does, and we appreciate you letting us know what you would like to see next! 

We’d love to hear from you again! Please use this [link](https://github.com/community/community/discussions/new?category=issues) to continue providing product feedback back in the meantime!
